.github/workflows/ci.yml

```diff
@@ -96,9 +96,15 @@ jobs:
       - name: Install cargo-audit
         run: cargo install --debug --locked cargo-audit
 
-      - name: Check dependencies 
+      - name: Check dependencies
         run: cargo audit
```

Comment:
Is there a reason to keep `cargo audit` if `cargo check advisories` already covers security advisories?

Reply:
Great question. I primarily wanted to use `cargo deny` in order to check licenses. I wasn't sure if it did exactly the same thing as `cargo audit`, so I thought it would be good to have them both run side-by-side for a bit, and then remove `cargo audit` in due course. But by the looks of it, yes it does the same things as `cargo audit`!

Reply:
@gguillemas we can remove `cargo audit` now if that is better?

Reply:
Got it. The main reason I added `cargo audit` was to fail the build and "force" somebody to update the affected dependency as soon as possible. Since we already have Dependabot double-checking dependencies, I would remove `cargo audit` already and see how `cargo deny` works by itself.

Reply:
Ok cool. Yes `cargo deny` will fail the build, and show the advisory (unless specifically allowed like the one we have there at the moment).

Ok I'll remove it now 👍 .

Reply:
Done!